### PR TITLE
Add Unit tests to pkg/cluster

### PR DESCRIPTION
**Pull request by qbarrand**

### CODE REVIEW

**Comment by qbarrand** [comment]
/cc @pacevedom @yevgeny-shnaidman @pmtk
PTAL

File: pkg/cluster/cluster_test.go [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+				})
+
+				return uo
+			}(),
+		}
+
+		o0, o1, o2, err := cluster.NewCluster(nil).OperatingSystem()
+		Expect(err).NotTo(HaveOccurred())
+		Expect(o0).To(Equal("rhel1"))
+		Expect(o1).To(Equal("rhel123"))
```
Comment (pacevedom) [inline pkg/cluster/cluster_test.go:340]: While this test is correct, not sure about the function's logic now. rhel version is different than appending major and minor.

**Review by pacevedom — APPROVED** [review]
minor comment

**Pull request merged**